src/modules/package.rs

```diff
@@ -0,0 +1,89 @@
+use super::Segment;
+use crate::context::Context;
+use ansi_term::Color;
+use serde_json;
+use std::fs::File;
+use std::io::Read;
+use std::path::PathBuf;
+use toml;
+
+/// Creates a segment with the current package version
+pub fn segment(context: &Context) -> Option<Segment> {
+    match get_package_version(context) {
+        Some(package_version) => {
+            const RUST_CHAR: &str = "📦";
+            const SEGMENT_COLOR: Color = Color::Red;
+
+            let mut segment = Segment::new("package");
+            // TODO: Should have prefix "is "
+            segment.set_style(SEGMENT_COLOR);
+
+            segment.set_value(format!("{} {}", RUST_CHAR, package_version));
```

Comment:
You can chain these, i.e. `segment.set_style(...).set_value(...)`

Reply:
@sirMerr This would over complicate the line ... I'm leaving it as is!